segmentation/utils/data_utils_test.py

```diff
@@ -104,6 +120,49 @@ def test_load_imgs_from_mibitiff():
         data_xr.values[0],
         (tiff.read(os.path.join(data_dir, mibitiff_files[0])))[channels].data)
 
+    # this is looped to make sure temp_file doesn't have '_' in its name
+    # (sadly there's no excluded characters option for tempfile)
+    while True:
+        with tempfile.NamedTemporaryFile(suffix=".tif", dir=data_dir) as temp_file:
```

Comment:
I did some googling, you're right. 

What about renaming it instead? os.rename

Reply:
I could try it, but I think I would have to handle cleanup manually then.  Alternatively, I could create a temporary directory, and create two mibi tiffs (or copy one in input_data) to test in it, and then just delete the directory.  Now that I type it out, that seems better than literally gambling on no underscores, so I'll probably do that.